files/initramfs-tools/arista-convertfs.j2

```diff
@@ -82,6 +82,24 @@ run_cmd() {
     fi
 }
 
+create_varlog_file() {
```

Comment:
for onie installer, it is in the installation process. For arista, since it does not have installation process, is needs to be put here. 

I think the varlog filesystem should be created in the installation process. For arista platform, I am also asking them to push these operations into their installation process (they are coming up with some solution there). Thus, moving to union-mount does not look like to be long term solution.